pkg/roachprod/cloud/cluster_cloud.go

```diff
@@ -346,14 +355,44 @@ func CreateCluster(l *logger.Logger, opts []*ClusterCreateOpts) error {
 			p = (p + 1) % providerCount
 		}
 
+		var vms vm.List
+		var vmsLock syncutil.Mutex
```

Comment:
Nitty nit: it seems in other places we already use `vmList` and `vmListMutex`. No strong preference, hence a nit :)

Reply:
Already add chosen `vmList` for the variable name inside the goroutine, so chose another one outside of it... updated to go the other way around, it's more consistent :)